### ISSUE TITLE
v2 Read Frame level data

### ISSUE THREAD

**Issue by atanasko** [waymo-research/waymo-open-dataset#643]
I try to read the data in a Frame, as in the [example](https://github.com/waymo-research/waymo-open-dataset/blob/master/tutorial/tutorial_v2.ipynb), but as in issue #628, I can also not read the VEHICLE bb. Is it possible (and if possible,  how) to read VEHICLE bb when retrieving data using "camera_to_lidar_box_association"?

**Comment by alexgorban** [comment]
No, association labels are available only for pedestrians at the moment.

**Comment by atanasko** [comment]
Thanks @alexgorban! 

What will be a good approach to retrieve data on a "Frame" level for camera and lidar (VEHICLE bb are there when camera data retrieved via 'camera_image' and 'camera_box' data frames merge)?
